Ternary Search Tree/TST.playground/Contents.swift

```diff
@@ -25,18 +25,18 @@ var testStrings: [(key: String, data: String)] = []
 let testCount = 30
 for _ in (1...testCount) {
     let randomLength = Int(arc4random_uniform(10))
-    let key = randomAlphaNumericString(randomLength)
-    let data = randomAlphaNumericString(randomLength)
+    let key = randomAlphaNumericString(withLength: randomLength)
+    let data = randomAlphaNumericString(withLength: randomLength)
 //    print("Key: \(key) Data: \(data)")
 
     if key != "" && data != "" {
         testStrings.append((key, data))
-        treeOfStrings.insert(data, withKey: key)
+        treeOfStrings.insertData(data, withKey: key)
```

Comment:
Would be cool if this function parameter would read:

`treeOfStrings.insert(data: data, withKey: key)`